src/practices/LanguageIndependent/ReadmeCorrectlySetPractice.ts

```diff
@@ -0,0 +1,97 @@
+import { PracticeEvaluationResult, PracticeImpact } from '../../model';
+import { DxPractice } from '../DxPracticeDecorator';
+import { PracticeContext } from '../../contexts/practice/PracticeContext';
+import { PracticeBase } from '../PracticeBase';
+import { ReportDetailType } from '../../reporters/ReporterData';
+import { FixerContext } from '../../contexts/fixer/FixerContext';
+
+@DxPractice({
+  id: 'LanguageIndependent.ReadmeCorrectlySet',
+  name: 'Set README.md Correctly',
+  impact: PracticeImpact.high,
+  suggestion: 'Provide necessary sections in README',
+  reportOnlyOnce: false,
+  url: 'https://github.com/DXHeroes/knowledge-base-content/blob/master/practices/readme.md',
+  dependsOn: { practicing: ['LanguageIndependent.ReadmeIsPresent'] },
+})
+export class ReadmeIsCorrectlySet extends PracticeBase {
+  private parsedReadme: string[] = [];
+  private readmePath = 'Readme.md';
+
+  async isApplicable(): Promise<boolean> {
+    return true;
+  }
+
+  async evaluate(ctx: PracticeContext): Promise<PracticeEvaluationResult> {
+    if (!ctx.root.fileInspector) {
+      return PracticeEvaluationResult.unknown;
+    }
+
+    const parseReadme = (readmeFile: string) => {
```

Comment:
IMHO, it should be a private method, not a method in the method.

Reply:
Ohh will change this then